test_definitions.py

```diff
@@ -49,6 +50,9 @@ def __init__(self, path: Path):
 ]
 OTHER_DEFINITION_IDS = [d.id for d in OTHER_DEFINITIONS]
 
+DEFINITION_SCHEMA = json.load(open("definition_schema.json"))
+YAML_VALIDATOR = jsonschema.Draft202012Validator(DEFINITION_SCHEMA)
```

Comment:
Any particular reason why these are constants rather than local variables in the only test that references them?

Reply:
Changed to fixture in [88a4d0b](https://github.com/eiffel-community/eiffel/pull/402/commits/88a4d0ba31a8b069a6c5c7ed253366996dae36fe). Didn't want to include it in the function as I didn't see any point in creating a separate validator for each test run.